src/static-data/rules/rule_set_templates/__nds_kita_szenarien.yml

```diff
@@ -0,0 +1,26 @@
+# laut beschluss vom 4. mrz (kita in nds)
+
+__nds_kita_szenarien:
+      
+  - conditions:
+      date_from: '2021-03-08'
+      incidence_to: 100
+      incidence_days: 3
+    references:
+      - date: '4.3.2021'
+        link: 'https://www.mk.niedersachsen.de/startseite/aktuelles/presseinformationen/kitas-offnen-am-8-marz-in-festen-gruppen-schrittweise-weitere-schuloffnungen-ab-15-marz-tonne-grosser-schritt-nach-vorne-bei-den-kleinsten-bis-ostern-besuchen-alle-schulerinnen-und-schuler-ihre-schule-198040.html'
+    rules:
+      - text: "Szenario B: **Alle** Kinder dürfen wieder in die Betreuung. Es gelten ggf. Einschränkungen beim bringen & holen (je Einrichtung geregelt)"
+        status: limit
+
+  # Nur Betreuung für Kinder mit Eltern in systemrelevanten Berufen  
+  - conditions:
+      date_from: '2021-03-08'
+      incidence_from: 100
+    references:
+      - date: '4.3.2021'
+        link: 'https://www.mk.niedersachsen.de/startseite/aktuelles/presseinformationen/kitas-offnen-am-8-marz-in-festen-gruppen-schrittweise-weitere-schuloffnungen-ab-15-marz-tonne-grosser-schritt-nach-vorne-bei-den-kleinsten-bis-ostern-besuchen-alle-schulerinnen-und-schuler-ihre-schule-198040.html'
+    rules:
+      - text: "Szenario C: **Notbetreuung** nur für Kinder mit Eltern in systemrelevanten Berufen"
```

Comment:
[limit] Nur **Notbetreuung** für Kinder mit Eltern in **systemrelevanten Berufen**
[closed] **Keine** Betreuung für Kinder mit Eltern in **nicht** systemrelevanten Berufen